### ISSUE TITLE
ERROR! conflicting action statements: shell, sudo

### ISSUE THREAD

**Issue by pattrickrice** [veggiemonk/ansible-ohmyzsh#3]
Hey there! Ran into an issue when making this a role:
```bash
ERROR! conflicting action statements: shell, sudo

The error appears to be in '/home/patrick/setup/roles/zsh/tasks/main.yml': line 6, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Backing up existing ~/.zshrc
  ^ here
```
Ansible version
```bash
patrick@desktop:~/setup$ ansible-playbook --version
ansible-playbook 2.9.6
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/patrick/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.8.2 (default, Apr 27 2020, 15:53:34) [GCC 9.3.0]
```

**Comment by veggiemonk** [comment]
Hi @pattrickrice , sorry I haven't used this in a while and I have very little time these days.
Could you help by making a pull request?

**Comment by veggiemonk** [comment]
fixed by #4
